src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
You can consider wrapping each `div` of class `stars` into a `section` tag to improve the semantics of your HTML. Using `section` tag would give a clear indication that each `div` is a standalone section in your webpage.